tests/IntegrationTests/SimulateOidc/Properties/launchSettings.json

```diff
@@ -33,5 +25,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     }
+  },
+  "iisSettings": {
```

Comment:
why do we have iis at all? Can't we just use kestrel?

Reply:
@jennyf19 and/or @jmprieur what are your thoughts on keeping or removing IIS?

Reply:
would be just for manual testing purposes.